### PR TITLE
feat(publishing): artifact registry publishing

### PR DESCRIPTION
**Pull request by chris-h-phillips**
Incorporates the new io.spinnaker.artifactregistry-publish plugin to rosco-web. This will publish the apt and rpm packages for rosco to the Google Artifact Registry

### CODE REVIEW

**Comment by chris-h-phillips** [comment]
depends on https://github.com/spinnaker/spinnaker-gradle-project/pull/174/files

File: rosco-web/rosco-web.gradle [hunk 1]
```diff
@@ -1,4 +1,5 @@
 apply plugin: 'io.spinnaker.package'
+apply plugin: 'io.spinnaker.artifactregistry-publish'
```
Comment (chris-h-phillips) [inline rosco-web/rosco-web.gradle:2]: I'm not 100% sure what the full process for release a new version of the gradle plugins is but that would have to happen and then bump the version in rosco also to find this new plugin

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -40,6 +40,14 @@ jobs:
           ORG_GRADLE_PROJECT_nexusPgpSigningPassword: ${{ secrets.NEXUS_PGP_SIGNING_PASSWORD }}
         run: |
           ./gradlew --info publishToNexus closeAndReleaseNexusStagingRepository
+      - name: Publish apt and rpm packages to Google Artifact Registry
```
Comment (dbyron-sf) [inline .github/workflows/release.yml:43]:
```suggestion
      - name: Publish apt packages to Google Artifact Registry
```

**Review by link108 — APPROVED** [review]

File: rosco-web/rosco-web.gradle (hunk 1 above)
Comment (dbyron-sf) [reply rosco-web/rosco-web.gradle:2]: This is all taken care of...

**Pull request merged**